src/index.html

```diff
@@ -15,8 +15,105 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <a
+          href="./index.html"
+          class="logo"
+        >
+          <img
+            class="logo__item"
+            src="./images/logo.png"
+            alt="Logo"
+            width="40px"
+            height="40px"
+          />
+        </a>
+        <nav class="header__nav">
+          <ul class="main-nav">
+            <li class="main-nav__item">
+              <a
+                class="main-nav__link is-active"
+                href="./index.html"
+              >
+                APPLE
+              </a>
+            </li>
+            <li class="main-nav__item">
+              <a
+                class="main-nav__link"
+                href="#"
+              >
+                SAMSUNG
+              </a>
+            </li>
+            <li class="main-nav__item">
+              <a
+                class="main-nav__link"
+                href="#"
+              >
+                SMARTPHONES
+              </a>
+            </li>
+            <li class="main-nav__item">
+              <a
+                class="main-nav__link"
+                href="#"
+                data-qa="hover"
+              >
+                LAPTOPS & COMPUTERS
+              </a>
+            </li>
+            <li class="main-nav__item">
+              <a
+                class="main-nav__link"
+                href="#"
+              >
+                GADGETS
+              </a>
+            </li>
+            <li class="main-nav__item">
+              <a
+                class="main-nav__link"
+                href="#"
+              >
+                TABLETS
+              </a>
+            </li>
+            <li class="main-nav__item">
+              <a
+                class="main-nav__link"
+                href="3"
```

Comment:
The `href` attribute for the 'PHOTO' link is set to '3', which is likely incorrect. It should be a valid URL or path. Please verify and correct this link.